packages/code-du-travail-data/Dockerfile

```diff
@@ -28,5 +22,9 @@ COPY ./dataset/suggestions.txt ./dataset/suggestions.txt
 
 WORKDIR /app
 ENV SUGGEST_FILE=../dataset/suggestions.txt
+
+RUN yarn -s --silent dump >! ./dist/dump.tf.json
+COPY ./dist/dump.tf.json /app/dump.tf.json
```

Comment:
why not dumping straight to `/app/dump.tf.json` ?

Reply:
what is `>!` ?

Reply:
I don't know why but, zsh ask me a confirmation when the file already exist -__-'